fppanalysis/two_dim_velocity_estimates.py

```diff
@@ -274,6 +298,7 @@ def estimate_velocities_for_pixel(
         y: pixel index y
         ds: xarray Dataset
         method: 'cross_corr' or 'cond_av'
+        naive: [bool] If True, use 1D method to estimate velocities. If False, use 2D method.
```

Comment:
Style: We add this functionalty only for comparison. This function is meant to be used with naive=False. 
However, for someone who is using the code for the first time, it might not be clear what naive means at once. I would change the name so that the default (True) is the improved method, and I would set to to True to default. For example, smth like use_2d_estimation.